modules/backend/widgets/Form.php

```diff
@@ -711,6 +711,18 @@ public function addFields(array $fields, $addToArea = null)
             }
         }
     }
+    /**
+     * Add tab icons
+     *
+     * @param array $icons
+     * @return void
+     */
+    public function addTabIcons(array $icons)
+    {
+        $this->allTabs->primary->icons = $icons;
+        $this->allTabs->secondary->icons = $icons;
+        $this->allTabs->outside->icons = $icons;
```

Comment:
@Samuell1 Will assigning the same icons array to all the tabs result in, for example, a primary tab called "Colours" and secondary tab called "Colours" having the same icon?

Reply:
Yes, but there is 0% chance that somebody wants to add same tabs names for theme customization.

Reply:
@Samuell1 I wouldn't be so sure - for example, someone has a primary tab called Colours to edit the global colour scheme, and then a primary tab called Header, which has a secondary tab called Colours to handle just the colourization of the header element. Now both of those Colours tabs have the same icon.

Reply:
@bennothommo Yeah, we solved it with @LukeTowers with more robust way